python/eos/analysis_file_description.py

```diff
@@ -202,6 +202,7 @@ class ParameterComponent(Deserializable):
     central: float
     min: float
     max: float
+    alias_of: list = field(default_factory=list)
```

Comment:
This should be squashed into the commit introducing ``analysis_file_description.py``.

Reply:
Done